### DISCUSSION TITLE
Manage submodules does not show submodules

### DISCUSSION THREAD

**Discussion by egoltzman** [gitextensions/gitextensions#9391] [Q&A] [answered]
I'm using V3.5.1.
I have a repo that have several submodules, I handled them from Manage Submodules window.
From some reason I can't see them anymore in this window, I can access each one by opening Git Extension in their local directory.
I can see them also in the left Submodules tree window.
Any idea?

**Comment by mstv** [comment, marked as answer]
You could check the `Git command log` (shortcut `F12`) for the result of `git submodule status`.
In addition, you can run the command manually, e.g.:
```
$ git submodule status
 0296f95158a7da669b44aaa264d8793c478ef648 Externals/EasyHook (v2.7.6789.0-10-g0296f95)
 3ebf19b8fc2ab8a24f54aab64921c88577fff05d Externals/Git.hub (remotes/origin/Fix_version-6-g3ebf19b)
 467710117fe6a1df2e96791b51f550a04fccba74 Externals/ICSharpCode.TextEditor (remotes/origin/UseArcade-4-g4677101)
 6fcd3f5b757e4b49aed8843141f619f5bd38e49d Externals/conemu-inside (remotes/origin/UseArcade-16-g6fcd3f5)
```

**Reply by egoltzman** [reply]
Thanks!
By running "git submodule status" I found out that I have a problem as I got this error:
`fatal: no submodule mapping found in .gitmodules for path 'XYZ'`
I then did this (After looking [here ](https://stackoverflow.com/a/13394710/6309)):
`git rm --cached ../XYZ`
And after that the "git submodule status" run smoothly and so does the submodule windows in Git Extensions
